content/07.pathogenesis.md

```diff
@@ -331,33 +330,32 @@ Contact, droplet, and aerosol transmission are therefore all worth evaluating wh
+As these trends became apparent, concerns arose due to the potential for individuals who did not yet show symptoms to transmit the virus [@url:https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html].
 Recovered individuals may also be able to transmit the virus after their symptoms cease.
 Estimates of the communicable period based on twenty-four individuals who tested positive for SARS-CoV-2 prior to or without developing symptoms estimated that individuals may be contagious for one to twenty-one days, but they note that this estimate may be low [@doi:10.1007/s11427-020-1661-4].
-Initially, viral nucleic acids were reported to remain at observable levels in the respiratory specimens of recovering hospitalized COVID-19 patients for a median of 20 days and with a maximum observed duration through 37 days, when data collection for the study ceased [@doi:10/ggnxb3].
+In an early study, viral nucleic acids were reported to remain at observable levels in the respiratory specimens of recovering hospitalized COVID-19 patients for a median of 20 days and with a maximum observed duration through 37 days, when data collection for the study ceased [@doi:10/ggnxb3].
 As more estimates of the duration of viral shedding are released, they are beginning to converge around approximately three weeks from first positive PCR test and/or onset of symptoms (which, if present, are usually identified within three days of the initial PCR test).
-For example, viral shedding was reported for up to 28 days following symptom onset by a second study [@doi:10.1038/s41586-020-2196-x] and for one to 24 days from first positive PCR test with a median of 12 days [@doi:10.1001/jama.2020.3204].
-On the other hand, almost 70% of patients were reported to still have symptoms at the time that viral shedding ceased, although all symptoms reduced in prevalence between onset and cessation of viral shedding (CVS) [@doi:10.1016/j.jcv.2020.104577].
-They also reported that the median time that elapsed between the onset of symptoms and CVS was 23 days and between first positive PCR test and CVS was 17 days [@doi:10.1016/j.jcv.2020.104577].
+For example, in later studies, viral shedding was reported for up to 28 days following symptom onset [@doi:10.1038/s41586-020-2196-x] and for up to 24 days from first positive PCR test, with a median of 12 days [@doi:10.1001/jama.2020.3204].
+The median time that elapsed between the onset of symptoms and cessation of viral RNA shedding (CVS) was 23 days and between first positive PCR test and CVS was 17 days [@doi:10.1016/j.jcv.2020.104577].
```

Comment:
The CVS acronym was removed in #790.  Will have to clean that up in a merge commit.

Reply:
I'll keep an eye out, thanks for pointing this out!